daily-patch-information/src/main/java/org/wso2/engineering/efficiency/patch/analysis/constants/Constants.java

```diff
@@ -0,0 +1,338 @@
+/*
+Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+WSO2 Inc. licenses this file to you under the Apache License,
+Version 2.0 (the "License"); you may not use this file except
+in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+package org.wso2.engineering.efficiency.patch.analysis.constants;
+
+import com.google.api.services.gmail.GmailScopes;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * constant values.
+ */
+public final class Constants {
+    // config file
+    public static final String CONFIG_FILE_PATH = "./config.properties";
+    public static final String DB_USERNAME = "dbUser";
+    public static final String DB_PASSWORD = "dbPassword";
+    public static final String PMT_CONNECTION = "pmtConnection";
+    public static final String JIRA_AUTHENTICATION = "JIRABasicAuth";
+    public static final String EMAIL_SENDER = "emailUser";
+    public static final String EMAIL_TO_LIST = "toList";
+    public static final String EMAIL_CC_LIST = "ccList";
+    public static final String URL_TO_JIRA_FILTER_CUSTOMER = "UrlToCustomerIssuesFilter";
+    public static final String URL_TO_JIRA_FILTER_INTERNAL = "UrlToInternalIssuesFilter";
+
+    public static final String GRAY_BACKGROUND = "#efefef";
+    public static final String WHITE_BACKGROUND = "#ffffff";
+
+    //SQL statements
+    public static final String SELECT_PATCHES_FOR_JIRA = "SELECT *, \n" +
```

Comment:
For a different type of constants, I think it's will be better if you put an inner class enclosing these. For example, you can have SQLStatement Class inside the Constants class, and then have all relevant constants within that. Please do that for all following constants.